src/framework/ui/card/cardHeader.component.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+  View,
+  ViewProps,
+  StyleProp,
+  ViewStyle,
+} from 'react-native';
+import {
+  ListItem,
+  ListItemProps,
+} from '../list/listItem.component';
+
+interface ComponentProps {
+  headerStyle?: StyleProp<ViewStyle>;
+  accentStyle?: StyleProp<ViewStyle>;
+}
+
+export type CardHeaderProps = Partial<ListItemProps> & ViewProps & ComponentProps;
+export type CardHeaderComponentElement = React.ReactElement<CardHeaderProps>;
+
+/**
+ * Styled `CardHeader` component can be used like `header` in the `Card` component.
+ *
+ * @extends React.Component
+ *
+ * @property {StyleProp<ViewStyle>} accentStyle - Determines style of the stripe element.
+ *
+ * @property {Partial<ListItemProps>}
+ *
+ * @property ViewProps
```

Comment:
```suggestion
 * @property ViewProps - Any props applied to View component.
```

Reply:
Since you apply `restProps` to ListItemElement, it should be `TouchableOpacityProps`